### PR TITLE
Xrddev 1050 e2etests keyscerts create

### PR DESCRIPTION
**Pull request by olliru**
Added tests
XRDDEV-1050 create key and csr
XRDDEV-1399 delete csr

### CODE REVIEW

File: src/proxy-ui-api/frontend/tests/e2e/pages/tabs/keysTab.js [hunk 1]
```diff
@@ -39,6 +39,246 @@ var keysTabCommands = {
...
+    return this;
+  },
+};
+
+var confirmationDialogCommands = {
+  confirm: function () {
+    this.api.keys(this.api.Keys.TAB);
+    this.api.keys(this.api.Keys.TAB);
+    this.api.keys(this.api.Keys.SPACE);
+    //this.click('@yesButton');
```
Comment (Riippi) [inline src/proxy-ui-api/frontend/tests/e2e/pages/tabs/keysTab.js:126]: Commented lines should be removed?
Also 3 more below this.

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-keyscerts-authsignkeys.js [hunk 2]
```diff
@@ -0,0 +1,718 @@
...
+    browser.waitForElementVisible(addKeyWizardDetails);
+
+    addKeyWizardDetails.enterLabel('');
+    addKeyWizardDetails.next();
+
+    browser.waitForElementVisible(addKeyWizardCSR);
+    addKeyWizardCSR.selectUsageMethod('AUTHENTICATION');
+    addKeyWizardCSR.selectService('X-Road Test CA CN');
+    addKeyWizardCSR.selectFormat('PEM');
+    //addKeyWizardCSR.selectClient('REST-UI-TEST:GOV:0245437-2');
```
Comment (Riippi) [inline src/proxy-ui-api/frontend/tests/e2e/specs/ss-keyscerts-authsignkeys.js:329]: Commented lines should be removed?

**Review by Riippi — CHANGES_REQUESTED** [review]

File: src/proxy-ui-api/frontend/tests/e2e/specs/ss-keyscerts-authsignkeys.js (hunk 2 above)
Comment (olliru) [reply src/proxy-ui-api/frontend/tests/e2e/specs/ss-keyscerts-authsignkeys.js:329]: Yes this line should be removed

**Comment by olliru** [comment]
Changes merger as part of another PR

**Pull request closed without merging**